ssm_parameter/main.tf

```diff
@@ -19,3 +29,19 @@ resource "aws_ssm_parameter" "ssm_parameter" {
   overwrite   = true
   tags        = var.common_tags
 }
+
+resource "aws_ssm_parameter" "random_ssm_parameter" {
```

Comment:
What does this do? Is is it creating a map off all the ssm params, so no need to keep calling the ssm paramters to retrieve values individually?

Reply:
The module does create multiple parameters which are then sent to the output as a map which you can get the parameter from by name. There's two different resources in here because for some of the parameters, the values are based on other resources and some are random like passwords. For the random ones, we need to set ignore value otherwise it will get overwritten each time we run terraform.